### PR TITLE
MM-47396 - trial ended banner not dismissable

### PR DESCRIPTION
**Pull request by pvev**
#### Summary
This PR adds the code to make the license expired banner dismissable.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-47396

#### Related Pull Requests
n/a

#### Release Note
```release-note
NONE
```

### CODE REVIEW

File: components/announcement_bar/renewal_link/renewal_link.tsx [hunk 1]
```diff
@@ -16,6 +16,7 @@ import {
 } from 'utils/constants';
 
 import NoInternetConnection from '../no_internet_connection/no_internet_connection';
+
```
Comment (pvev) [inline components/announcement_bar/renewal_link/renewal_link.tsx:19]: changes on this file were just remove unnecessary type React.FC from component, and remove unnecessary fragment surrounding component content.

**Review by salmanmanekia — APPROVED** [review]
Thanks @pvev LGTM! :shipit:

**Review by JulienTant — APPROVED** [review]
:shipit:

**Review by stevemudie — APPROVED** [review]
Worked as expected @pvev!

**Comment by amyblais** [comment]
@pvev This needs a manual cherry-pick to ``release-7.1`` branch (Maria had requested backporting this to the latest ESR).

**Comment by pvev** [comment]
> release-7.1

ok, on it, will let you know as soon as it gets ready!

**Comment by pvev** [comment]
> > release-7.1
> 
> ok, on it, will let you know as soon as it gets ready!

done https://github.com/mattermost/mattermost-webapp/commits/release-7.1

**Pull request merged**